lib/rex/ui/text/input/readline.rb

```diff
@@ -82,6 +82,13 @@ def gets()
       end
     end
 
+    def getch()
+      begin
+        self.fd.getch
+      rescue ::Errno::EINTR
+        retry
```

Comment:
Just confirming, is this the behavior we want? 🤔

Reply:
Potentially not, but it _is_ consistent with what we already have
https://github.com/rapid7/metasploit-framework/pull/15522/files/84003678af354993c7c83f95840c885914626226#diff-c323d311fe972840e9920c1bffaaf2746018bf5faa463c2ad1672ba18e19f0dfR77-R83